### PR TITLE
Add Select Subscriptions button

### PR DESCRIPTION
**Pull request by StephenWeatherford**
See Microsoft/vscode-cosmosdb#351

@fiveisprime, @chrisdias, @ealsur 
After discussing with Matt, I've done two things in the Storage extension, and will repeat for the others:

1) Added a "key" button on subscription nodes (tooltip: "Select Subscriptions...")
image

2) Added the following information to the readme:
image

### CODE REVIEW

**Comment by ealsur** [comment]
Awesome!! Thanks!!

File: README.md [hunk 1]
```diff
@@ -20,6 +22,20 @@ The Azure Storage extension for VS Code lets you browse Azure Blob Containers, F
 * `azureStorage.file.showSavePrompt`: Set to `false` to prevent showing a warning dialog on File file save.
 * `azureStorage.blob.showSavePrompt`: Set to `false` to prevent showing a warning dialog on Blob file save.
 
+## Signing In and Out of Azure and Filtering Subscriptions
```
Comment (EricJizbaMSFT) [inline README.md]: How about "Managing Azure Subscriptions"

File: README.md [hunk 2]
```diff
@@ -20,6 +22,20 @@ The Azure Storage extension for VS Code lets you browse Azure Blob Containers, F
...
+
+If you are not signed in to Azure, you will see a "Sign in to Azure..." link. Alternatively, you can select "View->Command Palette" in the VS Code menu, and search for "Azure: Sign In".
+
+![Sign in to Azure](resources/signInScreenshot.png)
+
```
Comment (EricJizbaMSFT) [inline README.md]: The casing for this picture is incorrect. You've got 'signInScreenshot' here, but the file is 'SignInScreenshot'


screen shot 2018-02-12 at 2 25 15 pm

File: package.json [hunk 3]
```diff
@@ -52,6 +52,14 @@
 					"dark": "resources/dark/refresh.svg"
 				}
 			},
+			{
+				"command": "azureStorage.selectSubscriptions",
+				"title": "Select Subscriptions...",
+				"icon": {
+					"light": "resources/selectSubscriptions.svg",
```
[the remark this answers is not in the document]
Comment (StephenWeatherford) [reply package.json]: I like the filter icon better, too.  @fiveisprime Matt, unless I hear objections I'll go ahead and use the filter.

File: README.md (hunk 1 above)
Comment (StephenWeatherford) [reply README.md]: Done.

File: resources/selectSubscriptions.svg [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (EricJizbaMSFT) [inline resources/selectSubscriptions.svg]: The new icon looks pretty dark - does it show up in dark theme ok?

**Comment by StephenWeatherford** [comment]
Sorry, shouldn’t have pushed. It’s not the final one.

File: resources/selectSubscriptions.svg (hunk 4 above)
Comment (StephenWeatherford) [reply resources/selectSubscriptions.svg]: New icon:
image

**Pull request merged**